src/checksum/checksum.csproj

```diff
@@ -23,6 +24,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>

```

Comment:
Since this conditional property group is specifically for building x86, this property should probably not be here.

Reply:
I will get this fixed up.